nx-libs.spec

```diff
@@ -444,16 +444,18 @@ rm -r %{buildroot}%{_includedir}/nx-X11/Xtrans
 %dir %{_libdir}/nx
 %dir %{_datadir}/nx
 %{_datadir}/nx/SecurityPolicy
-%{_datadir}/nx/XErrorDB
-%{_datadir}/nx/Xcms.txt
+%{_datadir}/nx/X11/XErrorDB
+%{_datadir}/nx/X11/Xcms.txt
 
 %files -n libNX_X11-6
 %defattr(-,root,root)
 %{_libdir}/libNX_X11.so.6*
+%{_libdir}/libNX_X11.la
```

Comment:
I'm not sure we want to actually ship `.la` files. Debian and other distributions I know just delete them. Is Fedora/RHEL handling that differently?